action.yml

```
@@ -0,0 +1,25 @@
+# action.yml
+name: 'DCO check'
+description: 'A reusable action that checks that all commits for a proposed change are signed off'
+
+inputs:
+  args:
+    description: "Arguments to be added the dco_check.py execution line."
+    required: false
+    default: ''
+  python-version:
+    description: "The version of python to be used."
+    required: false
+    default: '3.12'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: Check DCO
+      run: python ${{ github.action_path }}/dco_check.py ${{ inputs.args }}
```

Unfortunately, it seems like it the new action action doesn't quite work: https://github.com/christophebedard/dco-check/actions/runs/10955090208/job/30451152152?pr=129#step:2:33

The path here might just be missing the `dco_check` directory:

```suggestion
      run: python ${{ github.action_path }}/dco_check/dco_check.py ${{ inputs.args }}
```